handler/transport/backend.go

```diff
@@ -557,7 +586,17 @@ func (b *Backend) Value() cty.Value {
 	b.healthyMu.RLock()
 	defer b.healthyMu.RUnlock()
 
-	return seetie.GoToValue(map[string]interface{}{
+	var tokens map[string]interface{}
+	for _, auth := range b.requestAuthorizer {
+		if name, v := auth.value(); v != "" {
```

Comment:
@johakoch returns token request name atm. Not sure how authorizedName is involved to this.
Also the mapping to `token` is still missing here. We should map `token` only if the `token_request` name is `default` instead of:  

> if it is just one, map to token

Reply:
Done